File: app/plantilla/src/app/demo/models/PostreViewModel.ts
export class Postre {
    Post_Id?: string;
    Post_Descripcion?: string;
    Post_Precio?: string;
    Post_Imagen?: string;
    Post_Usua_Creacion?: string;
    Post_Fecha_Creacion?: string;
    Post_Usua_Modifica?: string;
    Post_Fecha_Modifica?: string;
    Post_Estado?: string;
}

export class LlenarPostres{
    post_id: number;
    post_Descripcion: string;
    post_Precio: string;
    post_Imagen: string;
    usua_Creacion: string;
    usua_Modifica: string;
    post_Fecha_Creacion: string;
    post_Fecha_Modifica: string;
}

export class CargarPostres{
    post_id?: number;
    post_Descripcion?: string;
}

export class Postre2{
    post_id: number;
    post_Descripcion: string;
    post_Precio: string;
    post_Imagen: string;
    post_Usua_Creacion: number;
}

export class PostreActualizar{
    post_id: number;
    post_Descripcion: string;
    post_Precio: string;
    post_Imagen: string;
    post_Usua_Modifica: number;
}

export class dropPostre{
    value?:String;
    text?:String;
}

export class PostreEnviar {
    Post_Id?: string;
    Post_Descripcion?: string;
    Post_Precio?: string;
    Post_Imagen?: string;


}

export class Fill {
    post_Id?: string;
    post_Descripcion?: string;
    post_Precio?: string;
    post_Imagen?: string;
    post_Usua_Creacion?: string;
    post_Fecha_Creacion?: string;
    post_Usua_Modifica?: string;
    post_Fecha_Modifica?: string;
    post_Estado?: string;
}

File: api/Proyecto_BK/Proyecto_BK.Common/Models/PostreViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_BK.Common.Models
{
    public class PostreViewModel
    {

        public int Post_id { get; set; }
        public string Post_Descripcion { get; set; }
        public decimal? Post_Precio { get; set; }
        public string Post_Imagen { get; set; }
        public int? Post_Usua_Creacion { get; set; }
        public DateTime? Post_Fecha_Creacion { get; set; }
        public int? Post_Usua_Modifica { get; set; }
        public DateTime? Post_Fecha_Modifica { get; set; }
        public bool? Post_Estado { get; set; }
        [NotMapped]
        public string CantidadAgregada { get; set; }
        [NotMapped]
        public string Usua_Creacion { get; set; }
        [NotMapped]
        public string Usua_Modifica { get; set; }
        [NotMapped]
        public string text { get; set; }
        [NotMapped]
        public int value { get; set; }
        [NotMapped]
        public string image { get; set; }
        [NotMapped]
        public decimal precio { get; set; }

    }
}
